### DISCUSSION TITLE
Have I completely messed up?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#911] [Q&A]
I thought I was making great progress today.  Did a quick "dry fit" of the probes in the flow cell--screwed them in part way to make sure the threads got along.  All seemed fine.  So, I went ahead & calibrated the pH &  ORP probes, rinsed them, wrapped them with a fair amount of teflon tape and installed them in the flow cell.

Leaks galore on the top of the flow cell :-(  I uninstalled the probes, re-wrapped them with a fair amount of teflon tape again & no love.

I thought the leak(s) in part may have been from the flow switch.  I removed it for boring & tapping the probe holes.  Installed it with fresh tape on  the bench as I did with the input, output, & drain valves.  Nothing was "cranked down" and all seemed good.  I had not noticed any leaks previously, but the probe holes were vacant.  Certainly none of the valves were leaking.

Just to be sure, I removed the flow switch from the flow cell, re-wrapped it with tape, & went to re-install it.  For the life of me, I cannot get the little bugger to remain vertical as I install it (hands only--no tools ever used except to loosen it when I removed it originally).  Little bugger lists to the left & the flow cell "float" catches on the side.  Threads on the flow cell seem fine (a bit hard to see them whilst doing pad ballet) & the threads on the switch also seem ok.  

Is  there some magic to reinstalling the switch?  It was operating correctly when I was testing earlier this week without the probes installed, so I don't think it was cross-threaded.

It also looks like there may be some leakage under the teflon tape on one/both of the probes.  I'm being very cautious tightening them, as I don't want to crack the cell, assuming I haven't already damaged it.

So, I'm at a dead end inches from the finish line.

Any tips & wisdom from the flow cell gods are appreciated greatly!

**Comment by tagyoureit** [comment]
Generally, if water is getting out that means you've got a little room to go before it is too tight.  I've found I need to go tighter than I feel comfortable with (or live with a little drip).  

As for the float, sounds like something is cross-threaded.  A slight downward pressure while turning and keeping it centered may help correct it.

**Reply by FlaMike** [reply]
Thank you for the suggestions!  I am certainly OK with a small drip & ok with tightening *very* gently 1/4 turn at most to slow it.  I am thinking of removing the pH probe, closest to the flow switch, to get more direct access when installing it.  Fingers crossed & thanks again!

**Reply by FlaMike** [reply]
The good news is that I removed & re-taped the pH & ORP probes with a boatload of tape.  I reinstalled them & screwed them noticeably further into the flow cell than before.  They are either not leaking or it's just a trivial amount, so they're in pretty good shape at the moment.

And now for the not-so-good news.  No love on the flow switch.  I removed it & reexamined the threads using better light & a magnifying glass.  It appears that a small portion of one, obviously critical, thread has some slight damage.  I truly don't understand how & when I inflicted it, but it's there.  In spite of numerous attempts to reinstall the flow switch vertically, after about 1-2 turns it takes a hard left that I don't seem to be able to overcome.  Although I'm not able to see/feel all of the threads on the flow cell I don't see to see any damage on them.  Hoping that's actually the case!

In the absence of any better ideas on my part, I'm going to order another flow switch & hope I can install it successfully.

**Reply by tagyoureit** [reply]
I'd grab a 1/2" male (metal, if possible) fitting and try to use that to re-align the threads by turning the float switch threads against the metal threads.  Would be a shame to have to buy a new one, but that might ultimately be the answer.

**Reply by FlaMike** [reply]
I assume it's a 1/2" NPT fitting I'd need.  I was wondering about such a possibility.  For better or worse, I've already ordered a new switch & it has "shipped" which likely means I can't cancel the order.  Fortunately, poolblu.com had the switch for $30.00, plus shipping & handling, of course.  Far less expensive than most places (over $100.00 is common for the switch).  For the prices some places charge you could get a whole new flow cell with the switch.  Hope if doesn't come to that!!

Nevertheless, I appreciate your suggestion.  However it works out, I may get a fitting & try to realign the threads lest I mess up another flow switch.  The joys of over-priced pool equipment--and  less than fully competent folks like me.

**Reply by FlaMike** [reply]
I think I'll give this one a shot--looks like  it may be  able to help re-align both the switch and the flow cell threads if necessary.

https://www.amazon.com/dp/B0BNTHFS59?ref=emc_s_m_5_i_n

**Reply by FlaMike** [reply]
Last night I received the brass fitting and after wrestling with the switch for 30-40 minutes was able to thread it *centered* in the fitting & then the flow cell.  I turned on the inflow & outflow valves on the cell & the switch responded as expected.  So, as  @tagyoureit suggested, I  should have been a bit more patient before ordering a replacement switch.  Nevertheless, it will arrive Thursday & I'll hang onto it should a problem arise...  There is  a little bit of slow leakage on the flow cell, but nothing concerning.  I'm hoping I can eventually tweak the flow rates to keep everything happy & not create a pressure cell in the process.  I assume as  long as  the flow through the cell is sufficient to keep the switch happy, there's enough water going  past the probes to generate accurate readings.

I noticed the pH reading was fairly different than my Taylor test kit's value (about 6.9 for the pH probe and 7.4 for the Taylor kit).  Hoping I don't need to go through another round of calibration.  I also need learn about my pool's ORP responses to determine an appropriate target.  The ORP probe reports 742 mv and the Taylor test showed a Cl level of 1.6--a little low for my taste.  The water temperature was a bout 65 Fahrenheit when I took my samples.  I used the drain line on the flow cell to get my water for the Taylor tests so it would be as close as possible to the water the probes were reading.  At the time of testing, the salt level was about 3200 PPM, the TA 80 and the Ca level about 360.  I added a little bit of CYN to help stabilize the Cl level, but have not reached my "summer" level yet, it's probably about 30 ppm but I have not measured it yet.  The pool has a screen so I usually keep that level around 50 or a bit less.   I'm going to let things run manually without engaging REM CHEM control of the acid pump or Intellichlor just yet.

I turned off the flow cell this morning because we're having some work done on the pool deck & they're making a mess of the pool in the process.  I didn't care to clog the cell or the inline filter when I know I'll be vacuuming the pool for the next several days.  The pool chemistry will likely be a mess for several days once they finish & I'd like to get things balanced before switching over to full REM control.

Hoping I made two steps forward & only one step backwards so far this week.

As always, I appreciate your help & if I am doing something foolish or failing to do something smart, please let me know.  Still drinking from a fire hose on my learning curve.

**Reply by tagyoureit** [reply]
Good progress!  The best advice is find the measurements that make you (and your pool) happy and don't worry about the exact values.  I used to keep my pool at 7.2 ph and 650 orp.  The probes and the test kits all have a margin of error and they may never read the same.  Also, orp can swing quite a bit with temperature, sunlight, other chemicals, etc.  Track it for a few weeks and you'll get a sense of what "good" readings from your probes are pretty quickly.

**Reply by FlaMike** [reply]
Will do!

Thank you for your positive & helpful feedback--as always!  I will wait for my pool to recover from its current assault before engaging REM fully.  Looking forward to taking advantage of your creation!  You, @rstrouse, and your other compatriots are amazing & very much appreciated!!
